jobs/src/main/scala/dcos/metronome/utils/time/Clock.scala

```diff
@@ -1,28 +0,0 @@
-package dcos.metronome
-package utils.time
-
-import org.joda.time.{ DateTimeZone, DateTime }
-
-import scala.concurrent.duration.FiniteDuration
-
-trait Clock {
```

Comment:
I think I do like the idea of abstracting the clock... especially for metronome

Reply:
java.time.Clock is an interface, right? What else would you want to be abstracted?

Reply:
(Marathon used to have a custom abstract clock implementation, we merged it to java.time.Clock without losing any needed flexibility)